packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -145,76 +145,119 @@ <h2>
-                    </oui-inline-adder-field>
-                    <oui-inline-adder-field data-adaptive>
-                        <oui-field
-                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_permission' | translate  }}"
-                            size="m"
-                        >
-                            <label class="oui-select">
-                                <select
-                                    name="permission"
-                                    ng-model="permission"
-                                    class="oui-select__input"
-                                    required
-                                >
-                                    <option
-                                        ng-repeat="perm in $ctrl.volumesPermissions"
-                                        value="{{perm}}"
-                                    >{{::
-                                        'pci_projects_project_training_jobs_submit_data_permission_'
-                                    + perm | translate }}
-                                    </option
+                <oui-switch
+                    data-model="$ctrl.job.volumeCommand"
+                    name="volumeCommand"
+                >
+                </oui-switch>
+            </oui-field>
+            <div
+                class="oui-field__control"
+                data-ng-if="$ctrl.job.volumeCommand"
+            >
+                <div class="oui-inline-adder mb-2">
+                    <div
+                        class="oui-inline-adder__form"
+                        data-ng-class="{ 'oui-inline-adder__form_added': $ctrl.httpHeader[$index].added }"
+                        data-ng-repeat="httpHeader in $ctrl.httpHeader track by $index"
+                    >
+                        <fieldset class="oui-inline-adder__fieldset">
+                            <div class="oui-inline-adder__row">
+                                <div class="oui-inline-adder__field">
+                                    <oui-field
+                                        data-label="{{:: 'pci_projects_project_training_jobs_submit_data_container' | translate  }}"
                                     >
-                                </select>
-                                <span
-                                    class="oui-icon oui-icon-chevron-down"
-                                    aria-hidden="true"
-                                ></span>
-                            </label>
-                        </oui-field>
-                    </oui-inline-adder-field>
-                </oui-inline-adder-row>
-            </oui-inline-adder>
+                                        <oui-select
+                                            id="container"
+                                            name="container"
+                                            data-model="$ctrl.httpHeader[$index].model.container"
```

Comment:
```suggestion
                                            data-model="httpHeader.model.container"
```
Isn't possible ? (For all models)

Reply:
`httpHeader` is an array of objects with `index` as `key`, so it is required to map with `$index`